README.md

```diff
@@ -141,6 +141,7 @@ both for the command line and for other platforms:
 - [Visual Studio Code extension](https://github.com/bmuskalla/vscode-tldr) available on [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=bmuskalla.vscode-tldr)
 - Web clients:
     - [tldr.jsx](https://github.com/ostera/tldr.jsx): http://tldr.ostera.io/
+    - [tldr.finzzz.net](https://git.finzzz.net/tldr/): https://tldr.finzzz.net/
```

Comment:
```suggestion
    - [tldr.finzzz.net](https://git.finzzz.net/tldr/): https://tldr.finzzz.net
```

Reply:
also the above link? tldr.ostera.io has a `/` at the end...

Reply:
Yeah, we can remove the that `/` too.